lens/util/store.go

```diff
@@ -70,6 +70,10 @@ func (cs *CachingBlockstore) HashOnRead(enabled bool) {
 	cs.blocks.HashOnRead(enabled)
 }
 
+func (cs *CachingBlockstore) Flush(ctx context.Context) error {
```

Comment:
Does this satisfy a new interface requirement?

Reply:
Here is the Flush in datasotre: https://github.com/filecoin-project/lotus/blob/master/blockstore/blockstore.go#L114.

And [here](https://github.com/filecoin-project/lotus/blob/master/blockstore/blockstore.go#L17-L31) is the interface. Is it correct? thanks

Reply:
Yup looks good, thanks.